cypress/projects/basic/playroom.config.js

```diff
@@ -3,4 +3,5 @@ module.exports = {
   snippets: './snippets',
   outputPath: './dist',
   openBrowser: false,
+  iframeSandbox: true,
```

Comment:
Turns out this was breaking Cypress (makes sense because `sandbox` limits iframe access); so I moved it to `examples/seek-style-guide.playroom.config.js`.